Documentation/contributing/testing/e2e.rst

```diff
@@ -64,15 +64,15 @@ To run all of the runtime tests, execute the following command from the ``test``
 
 ::
 
-    ginkgo --focus="Runtime" -noColor
+    ginkgo -v --focus="Runtime" -noColor
```

Comment:
Also wondering why we put `-noColor` everywhere. Such a shame to run in a black and white world...

Reply:
Also wondering

Reply:
I find `-test.v` useful also, did you consider adding it?

Reply:
I didn't know about `-test.v`. What's the difference?

Reply:
It's equivalent of `go test -v`, so if you have anything that uses test logger etc, it will be included. I don't know what it'd mean for Cilium tests, but I just use it to be sure when I run any gingko suite.

Reply:
If it makes any difference for Cilium tests, then yes, we should probably add it.